lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -304,6 +327,93 @@ impl Client {
         })
     }
 
+    async fn handle_clipboard(
+        cgo_handle: CgoHandle,
+        write_stream: &mut RdpWriteStream,
+        x224_processor: Arc<Mutex<X224Processor>>,
+        clipboard: &mut Option<String>,
+        clipboard_function: ClipboardFunction,
+    ) -> ClientResult<()> {
+        match clipboard_function {
+            ClipboardFunction::RequestFormatList => {
+                Client::process_cliprdr(x224_processor, write_stream, |c| c.initiate_copy(&[]))
+                    .await?
+            }
+            ClipboardFunction::Update(data) => {
+                clipboard.replace(data);
+                Client::process_cliprdr(x224_processor, write_stream, |c| {
+                    c.initiate_copy(&[ClipboardFormat::new(ClipboardFormatId::CF_UNICODETEXT)])
+                })
+                .await?;
+            }
+            ClipboardFunction::RemoteCopy(data) => {
+                let format = data
+                    .iter()
+                    .find(|cf| {
+                        cf.id() == ClipboardFormatId::CF_UNICODETEXT
+                            || cf.id() == ClipboardFormatId::CF_TEXT
```

Comment:
If we no longer support `ClipboardFormatId::CF_TEXT` for `ClipboardFunction::Update`, can we still support it here?

Reply:
It seems we're also currently supporting `CF_OEMTEXT`, why was that dropped?

https://github.com/gravitational/teleport/blob/e19bfee84116c17390847a92f65dae92c6afbfb1/lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs#L211-L212

Reply:
This is the other side, we didn't support `CF_TEXT` when we sent data from Teleport to Windows, we did the other way around.
As for the `OEM_TEXT` did we really support that? I think it won't convert if it's not ascii. I can add it back, the question is do we want it